sweepai/core/vector_db.py

```diff
@@ -70,11 +71,21 @@ def parse_collection_name(name: str) -> str:
 
 
 def embed_texts(texts: tuple[str]):
-    logger.info(f"Computing embeddings for {len(texts)} texts")
-    vector = sentence_transformer_model.encode(
-        texts, show_progress_bar=True, batch_size=BATCH_SIZE
-    )
-    return vector.squeeze()
+    HUGGINGFACE_URL = os.getenv('HUGGINGFACE_URL')
+    if HUGGINGFACE_URL:
+        try:
+            response = requests.post(HUGGINGFACE_URL, data={'texts': texts})
```

Comment:
use this as a reference for the request:

curl https://uu149rez6gw9ehej.eu-west-1.aws.endpoints.huggingface.cloud/distilbert-sentiment \
	-X POST \
	-d '{"inputs": "Deploying my first endpoint was an amazing experience."}' \
	-H "Authorization: Bearer <Token>"